src/analyze/analyze.c

```diff
@@ -1297,19 +1300,49 @@ static int dump(int argc, char *argv[], void *userdata) {
                         "org.freedesktop.systemd1",
                         "/org/freedesktop/systemd1",
                         "org.freedesktop.systemd1.Manager",
-                        "Dump",
+                        "Dump2",
                         &error,
                         &reply,
                         NULL);
```

Comment:
i think it ould be nice to tweak this a tiny bit, and default to regular old Dump() use if we are talking to a remote systemd. Specifically, a simple check for `arg_transport == BUS_TRANSPORT_REMOTE` should make us default to classic Dump()

Reply:
If the goal is to avoid a dbus call that's bound to fail anyway, wouldn't it be better to check sd_bus_can_send() with SD_BUS_TYPE_UNIX_FD?